modules/em/src/DensityMap.cpp

```diff
@@ -1104,6 +1104,75 @@ DensityMap *DensityMap::get_cropped(const algebra::BoundingBox3D &bb) {
   return cropped_dmap.release();
 }
 
+DensityMap *DensityMap::get_cropped(Particles ps, double distance, bool inverse, bool keep_map_dimensions) {
+  
+  algebra::Vector3Ds coords;
+  algebra::Vector3D vox_cent;
+  double vox_dist; 
+  double dsq=distance*distance; // square of distance (for comparisons)
+  bool crop;
+  Pointer<DensityMap> cropped_map;
+
+  // Always good to have a list of coordinates
+  for (unsigned int i=0; i<ps.size(); ++i){
+    coords.push_back(core::XYZ(ps[i]).get_coordinates());
+  }
+  
+  if(inverse==true || keep_map_dimensions==true) {
```

Comment:
`==true` is redundant for booleans. You can just say `if (inverse  || keep_map_dimensions) {`.